web3.js/src/transaction.js

```diff
@@ -452,17 +452,15 @@ export class Transaction {
 
   /**
    * Serialize the Transaction in the wire format.
-   *
-   * The Transaction must have a valid `signature` before invoking this method
    */
-  serialize(): Buffer {
+  serialize(verifySignatures: boolean = true): Buffer {
```

Comment:
WDYT about the `verifySignatures` parameter?   A separate `partialSerialize()` method felt clunky.  Passing in a config object could be an alternative.

Reply:
I think a config object sounds good. I think this method could have both a `verifySignatures` param to allow skipping verification altogether as well as a `partialVerify` param to only verify the signatures that are present.

Reply:
done!